### ISSUE TITLE
[feature] Mehrere Tabs/Übersicht

### ISSUE THREAD

**Issue by despecial** [obama/adventskalender-dashboard#16]
Erstmal: coole Sache mit dem Addon. Danke für die Arbeit.

In den vergangenen Jahren habe ich immer Bookmarks in Ordnern gemacht, da ich merkte, wenn ich bis zu 10 Tabs offen habe, bin ich schneller. 1.) laden alle Seiten parallel und nicht hintereinander und ich kann zB eins ausfüllen, während ich bei einem anderen ggf. ein Spiel spielen muss.

Gäbe es ne Möglichkeit zu sagen: öffne 5 Tabs, dann die nächsten 5?
Oder eine Übersicht aller Links, die man markiert (checkbox oder so) und dann öffnen lässt?
Allgemein wäre ein Übersicht gut (was hat man heute schon geklickt, was nicht, etc.)

**Comment by obama** [comment]
wie fändes du es wenn sich mit dem dashboard ein neues browser fenster öffnet? sodass der gewinnspielkram in einem eigenen fenster wäre. 
ich glaube (habe es noch nicht genauer angeschaut) das würde es vereinfachen umzusetzen. 
aber du hast schon recht, dass die ladezeiten einen großen teil ausmachen.

**Comment by despecial** [comment]
Ja, getrennt finde ich auf jeden Fall gut. Man könnte so im Dashboard/Übersicht sehen. welche Links man bereits geklickt hat (am aktuellen Tag oder letztes Datum) und welche noch übrig sind.

Die aktuelle Vorgehensweise hat sowas von Fragebogen und das ist meist nervig, weil die Navigation so eingeschränkt ist. 

Es gibt zB den Fall, dass man vielleicht 100 Links hat, aber um 23 Uhr nach Hause kommt und nur die wichtigsten schnell machen möchte. Da hilft dann eine Übersicht auch wieder.

**Comment by obama** [comment]
bzgl schneller laden habe ich erstmal ein `<link rel="preload">` eingebaut, aber laut wireshark läd er nix vor :/ 
die anderen sachen habe ich bisher nicht bearbeitet.
